cmd/dlt/userrole/cmd.go

```diff
@@ -233,15 +234,18 @@ func buildCommands(roleName string, roleARN string, isLinked bool, awsClient aws
 		return "", err
 	}
 	if hasPermissionBoundary {
-		deletePermissionBoundary := fmt.Sprintf("aws iam delete-role-permissions-boundary \\\n"+
-			"\t--role-name %s",
-			roleName)
+		deletePermissionBoundary := awscb.NewIAMCommandBuilder().
+			SetCommand(awscb.DELETE_ROLE_PERMISSIONS_BOUNDARY).
+			AddParam(awscb.ROLE_NAME, roleName).
+			Build()
 		commands = append(commands, deletePermissionBoundary)
 	}
 
-	deleteRole := fmt.Sprintf("aws iam delete-role \\\n"+
-		"\t--role-name %s", roleName)
+	deleteRole := awscb.NewIAMCommandBuilder().
+		SetCommand(awscb.DELETE_ROLE).
+		AddParam(awscb.ROLE_NAME, roleName).
+		Build()
 	commands = append(commands, deleteRole)
 
-	return strings.Join(commands, "\n"), nil
+	return strings.Join(commands, "\n\n"), nil
```

Comment:
why the extra newline between commands in this case?

Reply:
All other commands had this, I guess it is to space them out. Just wanted to make it standard in relation to others, most commands have two new lines, e.g creates and updates. For some reason deletes only have a single, maybe should add that so all are standardized.